package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "@prisma/client": "^5.5.2",
+    "graphql-scalars": "^1.22.4",
+    "graphql-tag": "^2.12.6"
+  }
+}
```

Comment:
This might be why your code is failing. For one, you can completely remove this file since we use a separate `package.json` in the frontend folder vs in the backend folder. This was probably created if you can `npm install ...` from the root folder. In the future, run `yarn add ...` in the backend/frontend folder.